fitwsarah_frontend/src/models/FitnessService.js

```diff
@@ -21,16 +23,55 @@ function FitnessServiceList() {
     }
 
     return (
-        <Container>
-            <h2 style={{ textAlign: 'center', margin: '2rem 0' }}>Services & Prices</h2>
-            <Row xs={1} md={3} className="g-4">
-                {services.map((service) => (
-                    <Col key={service.serviceId}>
-                        <FitnessServiceCard fitnessService={service} />
-                    </Col>
-                ))}
-            </Row>
-        </Container>
+        <div className="main-page">
+        <header className="main-header">
+            <div className="logo">FITWSARAH</div>
+            <nav className="main-nav">
+                <a href="/about">About</a>
+                <a href="/login">Log In</a>
+                <a href="/signup" className="signup-btn">Sign Up</a>
+            </nav>
+        </header>
+
+        <section className="hero-section">
+            {/* Optionally, if you want to overlay text on the hero image, you can do so here */}
+        </section>
+
+        <section className="services-section">
+            <Container>
+                <h2>Services & Prices</h2>
+                <Row>
+                    {services.map(service => (
+                        <Col key={service.id} md={4}>
+                            <div className="service-card">
+                                <h3>{service.title}</h3>
+                                <p>{service.description}</p>
+                                <div className="price">{service.price}$</div>
+                                <button className="book-button">Book</button>
+                            </div>
+                        </Col>
+                    ))}
+                </Row>
+            </Container>
+        </section>
+
+        {/* Include other sections like 'appointments-section' and 'packages-section' as needed */}
+
+        <footer className="main-footer">
```

Comment:
Aren't we supposed to have more inside the footer?

Reply:
Yes, we are supposed to have the social media platform icon link that will be placed right above the footer. I haven't found the right images yet that would fit the theme the most. They will be added in the future sprints.